src/ts/client/grapesjs/keymaps.ts

```diff
@@ -0,0 +1,64 @@
+import {Editor, PluginOptions} from 'grapesjs'
+import {cmdPublish} from './PublicationUi'
+import {cmdOpenFonts} from '@silexlabs/grapesjs-fonts'
+import {cmdToggleBlocks, cmdToggleLayers, cmdToggleNotifications, cmdToggleSymbols} from './index'
+import {cmdTogglePages} from './page-panel'
+import {cmdOpenSettings} from './settings'
+
+// Utility functions
+
+const selectBody = (editor: Editor): void => {
+  const wrapper = editor.DomComponents.getWrapper()
+  editor.select(wrapper)
+}
+
+const toggleCommand = (editor: Editor, name: string): void => {
```

Comment:
I have a question: why this toggleCommand function? It doesn't work the same if you pass a command to km.add ?

I ask because toggleCommand takes a command and implements something similar to grapes' [stateful commands](https://grapesjs.com/docs/modules/Commands.html#stateful-commands) right?

Reply:
I had tried using only the command. It doesn't work as expected, as it will open panels without closing the other ones, mainly. You have to stop the command associated with a panel to close it.